src/utils.rs

```diff
@@ -262,6 +262,8 @@ pub fn validate_contract_deployed<S: StateReader>(
 //* Internal objects utils
 //* ----------------------------
 
+// TODO: Remove warning inhibitor when finally used.
+#[allow(dead_code)]
```

Comment:
I think you can remove this #[allow(dead_code)]

Reply:
Removed, along with a few others. I also replaced some `#[allow(unused)]` I found with `#[allow(dead_code)]` (when needed) and fixed the bugs that came with those changes.